community/modules/scheduler/schedmd-slurm-gcp-v6-controller/modules/slurm_files/scripts/resume.py

```diff
@@ -509,11 +510,13 @@ def _allocate_nodes_to_placements(nodes: List[str], excl_job_id:Optional[int], l
     if not (nodeset.enable_placement and valid_placement_node(model)):
         return no_pp
     
+    max_count = calculate_chunk_size(max_distance, model) if max_distance else PLACEMENT_MAX_CNT
```

Comment:
* Don't use `model`, but `nodeset` instead;
* Don't pass `max_distance`, extract it from `nodeset`;
* Don't handle `if max_distance is None` here, `calculate_chunl_size` should take care of it.

Reply:
Done